python/src/detectors/python-avoid-raw-sql/python-avoid-raw-sql_compliant.py

```diff
@@ -0,0 +1,13 @@
+#  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#  SPDX-License-Identifier: MIT-0
+
+# {fact rule=python-avoid-raw-sql@v1.0 defects=0}
+from django.db.models.expressions import RawSQL
+from django.http import request
+
+def compliant(queryset):
+    someparam = request.GET.get("name")
```

Comment:
I think the Django request comes as a parameter to the function. Applicable to all files.